native/java/org/jpype/manager/TypeManager.java

```diff
@@ -48,6 +48,9 @@
   public TypeAudit audit = null;
   private ClassDescriptor java_lang_Object;
   public Class<? extends Annotation> functionalAnnotation = null;
+  // For reasons that are less than clear, this object cannot be created
```

Comment:
Doesn't it make sense for the JVM to block the creation of new objects during shutdown?

Reply:
I would have assumed it would have run it immediately if we are in shutdown.  Preventing another hook means we generate an exception during a fragile time.  But maybe that I think about it differently.    It creates lots of threads and objects at shutdown.